packages/core/src/app/address/googleAutocomplete/AddressSelectorUk.ts

```diff
@@ -2,10 +2,14 @@ import AddressSelector from './AddressSelector';
 
 export default class AddressSelectorUK extends AddressSelector {
     getState(): string {
-        return this._get('administrative_area_level_2', 'long_name');
+        return '';
     }
 
     getStreet2(): string {
         return this._get('locality', 'long_name');
     }
+
```

Comment:
Should we implement methods for other fields as well?

Reply:
I understand no product requirements for other fields right now, so it should be good to use the parent class' methods.